Contribute.swift

```diff
@@ -0,0 +1,166 @@
+import Foundation
+
+let filemgr = NSFileManager.defaultManager()
+let folder = filemgr.currentDirectoryPath
```

Comment:
@CodaFi I can see the Hacker News headlines now! `Member of Swift Team at Apple Bullies Open Source Community Builder Robert Mozayeni For Creating A Constant That Is Only Used Once, As If Xcode's Pedantry Wasn't Already Enough"